src/block/defines.rs

```diff
@@ -9,12 +9,24 @@ pub const VIRTIO_BLK_T_IN: u32 = 0;
 pub const VIRTIO_BLK_T_OUT: u32 = 1;
 /// Flush request.
 pub const VIRTIO_BLK_T_FLUSH: u32 = 4;
+/// Discard request.
+pub const VIRTIO_BLK_T_DISCARD: u32 = 11;
+/// Write zeroes request.
+pub const VIRTIO_BLK_T_WRITE_ZEROES: u32 = 13;
 
 // Feature bits.
 /// Read-only device.
 pub const VIRTIO_BLK_F_RO: u64 = 5;
 /// Flush command supported.
 pub const VIRTIO_BLK_F_FLUSH: u64 = 9;
+/// Discard command supported.
+pub const VIRTIO_BLK_F_DISCARD: u64 = 13;
+/// Write zeroes command supported.
+pub const VIRTIO_BLK_F_WRITE_ZEROES: u64 = 14;
+
+/// If this bit is set, the targeted range should be unmapped
+/// (only valid for write zeroes command).
+pub const VIRTIO_BLK_WRITE_ZEROES_FLAG_UNMAP: u32 = 1;
```

Comment:
This should be called somehow such that it is clear that it represents a bit. How about `WRITE_ZEROES_UNMAP_BIT`? I don't think we need to prefix it with VIRITO_BLK because it does not map 1:1 to the defines in the virtio spec.

Reply:
It's not the position, but the least significant bit set. I'll make the comment clearer.

Reply:
Moved the definition to `DiscardWriteZeroes` and hopefully, updated with a clearer comment.